tests/unittests/OperatorTest.cpp

```diff
@@ -8087,7 +8090,28 @@ TEST_P(OperatorTest,
       25,
   };
 
-  EXPECT_TRUE(expected.isEqual(result, 0.02));
+  EXPECT_TRUE(expected.isEqual(result, allowedError));
+}
+
+/// Test Fused-RWQ-SLWS in where the weights are in Fp16, data
+/// inputs are UInt8FusedQTy.
+TEST_P(
+    OperatorTest,
+    FusedRowwiseQuantizedSparseLengthsWeightedSum_ConvertedFloat16_NoFusedConvert) {
+  CHECK_IF_ENABLED();
+  return testRowwiseQuantizedSparseLengthsSum_ConvertedFloat16(
+      bindings_, mod_, F_, EE_, 0.02,
+      /* useFP16Accumulation */ false,
```

Comment:
This is false for both current calls, and it's not even being used inside `testRowwiseQuantizedSparseLengthsSum_ConvertedFloat16`, so let's remove it.

Reply:
I actually intentionally kept this to make the test helper more "configurable". I'll remove it  for now :)